kong/db/dao/init.lua

```diff
@@ -22,6 +22,8 @@ local log          = ngx.log
 local fmt          = string.format
 local match        = string.match
 local run_hook     = hooks.run_hook
+local sha256       = utils.sha256_hex
+local LMDB_MAX_KEY_SIZE = 511 -- LMDB default max key size
```

Comment:
`sha256` and `LMDB_MAX_KEY_SIZE` are only used by `hash_by_length`,
so we could move them near `hash_by_length` and into a `do ... end` block.

Reply:
511 may not be a proper number, cache_key is "key:ws_Id", a key with 511 bytes may still generate a long key.

Perhaps we should change it to `256`?

Reply:
ok